knowledge-base/resize-images-radpdfprocessing.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Resizing Large Images to Fit in the PDF Page
+description: Learn how to adjust large images to fit within the PDF page dimensions using RadPdfProcessing, preserving the aspect ratio.
+type: how-to
+page_title: How to Resize Images for PDF Documents in RadPdfProcessing
+slug: resize-images-radpdfprocessing
+tags: radpdfprocessing, document processing, image to pdf, image resize, aspect ratio
+res_type: kb
+ticketid: 1656650
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.2.426| RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+When converting an image to a PDF file using [RadPdfProcessing]({%slug radpdfprocessing-overview%}), the resulting PDF may cut off parts of the image if its dimensions (e.g., 1700 x 1700 px) exceed the PDF page size. The goal is to:
```

Comment:
Maybe change the description to the more general approach: When "inserting an image" into a PDF file.